src/Backends/IF97/IF97Backend.h

```diff
@@ -44,14 +45,14 @@ class IF97Backend : public AbstractState  {
         this->_rhomass.clear();
         this->_hmass.clear();
         this->_smass.clear();
-        this->_phase = iphase_not_imposed; 
+        this->_phase = iphase_not_imposed;
```

Comment:
(I have auto format for trailing spaces... view with ?w=1 in the URL to filter out whitespace)